js-wixcode-sdk/$w/repeater/guides/The Lifecycle of Repeated Items.md

```diff
@@ -0,0 +1,132 @@
+---
+articleType: concept
+---
+# The Lifecycle of Repeated Items
+
+
+When you work with a [repeater](https://dev.wix.com/docs/velo/velo-only-apis/$w/repeater/introduction) in code, it's important to understand how the items in the repeater are created, updated, and deleted. 
+
+## Connect data
+
+A repeater's data is an array of objects, which you get or set using the repeater's `data` property. Each object in this array must have a unique `_id` property that links the object to a specific repeated item in the repeater. The `_id` value can include only alphanumeric characters and hyphens (`-`). Besides `_id`, each object in the data array can include any other properties you need.
+
+The repeater doesn't automatically connect data in the elements inside each repeated item. You control how to connect data in those elements by using the `onItemReady()`, `onItemRemoved()`, `forItems()`, and `forEachItem()` callback functions.
```

Comment:
I think the use of "populate" here is better. Then we are using "connect" for setting data and "populate" for displaying that data.

```suggestion
The repeater doesn't automatically populate its data into the elements inside each repeated item. You control how to populate the data into those elements by using the `onItemReady()`, `onItemRemoved()`, `forItems()`, and `forEachItem()` callback functions.
```